.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - abhijeetnarvekar
+  - package-ecosystem: "nuget"
+    directory: "/"
```

Comment:
Is this path correct @abhijeetnarvekar ?

Reply:
@Thilo-Acrolinx The path of packages file is inside project directory https://github.com/acrolinx/acrolinx-sidebar-demo-dotnet/blob/main/Acrolinx.Demo.Sidebar/packages.config

Reply:
@abhijeetnarvekar I updated the config, please check again:
https://github.com/acrolinx/acrolinx-sidebar-demo-dotnet/pull/7/commits/8b7cecdb0de1e3e40f7d3e2a6e5f113807636f1c